data/Cernusak_2006/data.csv

```diff
@@ -1,226 +1,226 @@
-species,Tree,date,sampling_period,burn_status,R25_stem,Q10,Photosynthesis (umol m-2 s-1),PARi (umol m-2 s-1),Stomatal conductance (mol m-2 s-1),Ci (umol mol-1),Transpiration (mmol m-2 s-1),position,leaf respiraton at 30degC,sapwood cross sectional area (cm2),leaf fresh weight (g),dry weight to fresh weight ratio,leaf dry weight (kg),SW area per leaf dry mass (cm2 kg-1),specific leaf area (m2 kg-1),SW area per leaf area (cm2 m-2),diameter,diameter under bark,bark_thickness,context
```

Comment:
Just checking you meant to rewrite the entire dataset?

Reply:
There are added replicate values on a few rows of data